### PR TITLE
Only include gene panel id in the AlterationCountByGene

### PR DESCRIPTION
**Pull request by zhx828**
In the big panel, the return data would be to big to return the detailed info

# What? Why?
Fix https://github.com/cBioPortal/cbioportal/issues/6337

### CODE REVIEW

File: model/src/main/java/org/cbioportal/model/AlterationCountByGene.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	private Integer totalCount;
 	private Integer numberOfSamplesProfiled;
     private BigDecimal qValue;
-    private List<GenePanel> matchingGenePanels;
+    private List<String> matchingGenePanelIds;
```
Comment (kalletlak) [inline model/src/main/java/org/cbioportal/model/AlterationCountByGene.java]: @zhx828 Is there a possibility for duplicate ids. If yes then better to use `Set`.

Comment (zhx828) [reply model/src/main/java/org/cbioportal/model/AlterationCountByGene.java]: The way the matchingGenePanelIds generated would avoid the duplication. But I agree the Set probably better here.

Comment (zhx828) [reply model/src/main/java/org/cbioportal/model/AlterationCountByGene.java]: @kalletlak updated.

**Comment by jjgao** [comment]
@zhx828 maybe this should go to master?

**Comment by zhx828** [comment]
@jjgao this needs the backend+frontend changes deploy at the same time. I don't think master should be the place.

**Comment by jjgao** [comment]
I see. Let's make it into the next release then.

We may need to find a way to do quick bug fixes that requires both frontend and backend just like this one. @inodb

**Comment by inodb** [comment]
@jjgao the process is the same:

1. disable auto deploy to netlify on frontend
2. tag frontend version
3. update frontend version in backend pom
4. tag backend
5. deploy new tagged docker image to k8s without frontend url set

**Pull request merged**